frontend/public/assets/js/gViz/visualizations/graph-canvas/setup.js

```diff
@@ -36,40 +36,40 @@ gViz.vis.graph.setup = function () {
                     case 'circle':
                       _var.context.moveTo(d.x + d.radius / _var.transform.k, d.y);
                       return _var.context.arc(d.x, d.y, d.radius / _var.transform.k, 0, 2 * Math.PI);
-                    case 'rect':
-                      _var.context.moveTo(d.x - d.radius / _var.transform.k, d.y - d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x + d.radius / _var.transform.k, d.y - d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x + d.radius / _var.transform.k, d.y + d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x - d.radius / _var.transform.k, d.y + d.radius / _var.transform.k);
-                      return _var.context.lineTo(d.x - d.radius / _var.transform.k, d.y - d.radius / _var.transform.k);
-                    case 'triangle-down':
-                      _var.context.moveTo(d.x - d.radius / _var.transform.k, d.y - d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x + d.radius / _var.transform.k, d.y - d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x, d.y + d.radius / _var.transform.k);
-                      return _var.context.lineTo(d.x - d.radius / _var.transform.k, d.y - d.radius / _var.transform.k);
-                    case 'triangle-up':
-                      _var.context.moveTo(d.x, d.y - d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x + d.radius / _var.transform.k, d.y + d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x - d.radius / _var.transform.k, d.y + d.radius / _var.transform.k);
-                      return _var.context.lineTo(d.x, d.y - d.radius / _var.transform.k);
-                    case 'diamond':
-                      _var.context.moveTo(d.x, d.y - d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x + d.radius / _var.transform.k, d.y);
-                      _var.context.lineTo(d.x, d.y + d.radius / _var.transform.k);
-                      _var.context.lineTo(d.x - d.radius / _var.transform.k, d.y);
-                      return _var.context.lineTo(d.x, d.y - d.radius / _var.transform.k);
+                    //case 'rect':
```

Comment:
Me incomoda um pouco esse tanto de código comentado. Se ele não é utilizado, o melhor é deletar. Se eventualmente ele for necessário, está no histórico do git.